docs/src/main/paradox/routing-dsl/directives/future-directives/onSuccess.md

```diff
@@ -4,7 +4,15 @@
 
 ## Signature
 
-@@signature [FutureDirectives.scala]($akka-http$/akka-http/src/main/scala/akka/http/scaladsl/server/directives/FutureDirectives.scala) { #onSuccess }
+```scala
+def onSuccess(unitFuture: Future[Unit]): Directive0
+def onSuccess(simpleTypeFuture: Future[T_0]): Directive1[T_0]
```

Comment:
I guess this could be just `T`, but :+1:

Reply:
You are right. Simplified